File: Domain/Pharmacy.cs
using System;

namespace Domain
{
    public class Pharmacy
    {
        public Guid Id { get; set; }
        
        public string ProductName { get; set; }

        public int ProductCode { get; set; }

        public string Category { get; set; }

        public string Country { get; set; }

        public string Manufacturer { get; set; }

        public string Prescription { get; set; }

        public int Mg { get; set; }

        public float Price { get; set; }

        public DateTime ModificationDate { get; set; }

        public int Quantity { get; set; }
    }
}

File: client-app/src/app/models/pharmacy.ts
import { PatientProfile } from "./profile";

export interface Pharmacy {
    id : string;
    productName : string;
    productCode : string;
    category : string;
    country : string ;
    manufacturer : string;
    prescription : string;
    mg : string;
    price : string;
    modificationDate : string;
    quantity : string;
    patientId: string;
    patient? : PatientProfile;
}
export interface PharmacyDto{
    productName : string;
    productCode : string;
    category : string;
    country : string ;
    manufacturer : string;
    prescription : string;
    mg : string;
    price : string;
    quantity : string;
}